ballet/eng/base.py

```diff
@@ -39,13 +45,17 @@ class SimpleFunctionTransformer(BaseTransformer):
     def __init__(self, func, func_args=None, func_kwargs=None):
         super().__init__()
         self.func = func
-        self.func_args = func_args if func_args else ()
-        self.func_kwargs = func_kwargs if func_kwargs else {}
-        self.func_call = funcy.rpartial(
```

Comment:
Does funcy.rpartial differ in implementation than the new code?

Reply:
should be identical. I realized it is against sklearn estimator API to set attributes within __init__ that are not estimator parameters. So at that point, I might as well define a real method.